terminal/tcell/cell_options.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 Google Inc.
+//
```

Comment:
Can you please also update the [CHANGELOG.md](https://github.com/mum4k/termdash/blob/master/CHANGELOG.md) file in this PR? We can list this as "Added" functionality in the Unreleased section.